.vitepress/theme/custom.css

```diff
@@ -0,0 +1,43 @@
+* {
+    -webkit-font-smoothing: antialiased;
+}
+
+.main img {
+    border-radius: 8px;
+}
+
+h1.styled {
+    font-size: 36px;
+    margin-top: 0 !important;
+    padding-top: 0;
+}
+
+.mt-80 {
+    margin-top: 80px;
+}
+.mb-80 {
+    margin-bottom: 80px;
+}
+
+.m-40 {
+    margin: 60px 0px;
+}
```

Comment:
This could be replaced with inline classes `mt-[80px]`, `mb-80px` and `my-[60px]`